website/content/docs/enterprise/lts.mdx

```diff
@@ -0,0 +1,192 @@
+---
+layout: docs
+page_title: Long-term support for Vault
+description: >-
+  Learn the details about long-term support for Vault Enterprise.
+---
+
+# Long-term support for Vault
+
+@include 'alerts/enterprise-only.mdx'
+
+Long-term support (LTS) eases upgrade requirements for installations that cannot
+upgrade frequently, quickly, or easily.
+
+## LTS in a nutshell
+
+<table>
+
+  <thead>
+    <tr>
+      <th style={{width:'30%', verticalAlign:'middle'}}>Question</th>
+      <th style={{verticalAlign: 'middle'}}>Answer</th>
+    </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      <a href="#who">Who should consider long term support?</a>
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+      Enterprise customers using Vault for sensitive or critical workflows.
+    </td>
+  </tr>
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      <a href="#what">What is long-term support?</a>
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+      Extended support for select Vault versions. HashiCorp supports LTS
+      versions for two years. In the first year, LTS Vault versions receive
+      feature and critical updates. In the second year, LTS Vault versions
+      continue receiving critical updates.
+    </td>
+  </tr>
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      <a href="#where">Where do I download the LTS binary?</a>
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+      You do not need to download a separate binary for LTS support. Just make
+      sure to select an LTS Vault version when you <a href="/vault/install">install</a>
+      or <a href="/vault/docs/upgrading">upgrade</a> your Vault instance.
```

Comment:
```suggestion
      You do not need to download a separate binary for LTS support. Just make
      sure to select an LTS Vault version when you <a href="/vault/install">install</a> 
      or <a href="/vault/docs/upgrading">upgrade</a> your Vault instance.
```

<br />
Looks like you may need a space between "install" and "or"?


![image](https://github.com/hashicorp/vault/assets/7660718/62f48f10-45fc-4e0e-a196-527baeb0e606)

Reply:
I thought I fixed that already >_< Our HTML parser is weird about whitespace and HTML links. You need to end the line on a non-linked word or add a non-breaking space for it to respect the whitespace.